gtfs2netexfr/src/main.rs

```diff
@@ -30,40 +30,41 @@ struct Opt {
     #[structopt(short, long, parse(from_os_str), default_value = ".")]
     input: PathBuf,
 
-    /// Output directory
+    /// Output directory.
```

Comment:
Strange behaviour. All the dots are not displayed. Only if there are multilines.

Reply:
Ah, indeed, I didn't notice and that is strange. Although, I would let them for at least consistency in the source file.